vendor/modules.txt

```diff
@@ -2501,7 +2501,9 @@ sigs.k8s.io/kustomize/pkg/transformers
 sigs.k8s.io/kustomize/pkg/transformers/config
 sigs.k8s.io/kustomize/pkg/transformers/config/defaultconfig
 sigs.k8s.io/kustomize/pkg/types
+# sigs.k8s.io/structured-merge-diff/v3 => sigs.k8s.io/structured-merge-diff/v3 v3.0.1-0.20200706213357-43c19bbb7fba
```

Comment:
not sure why this was here after update-vendors.sh

- [ ] make sure this belongs here or remove it/it's cause

Reply:
https://github.com/kubernetes/kubernetes/pull/93867 moved to v4. Perhaps there are a few references to v3 lingering in this PR?

Reply:
Found it, thanks.